src/ui/components/shared/Dialog.tsx

```diff
@@ -6,14 +6,31 @@ export type DialogPropTypes = HTMLProps<HTMLDivElement>;
 
 export function Dialog({ children, className, ...props }: DialogPropTypes) {
   return (
+    <>
     <div
       {...props}
       className={classNames("dialog flex flex-col items-center", className)}
       role="dialog"
       style={{ animation: "linearFadeIn ease 200ms", width: 400 }}
     >
-      {children}
+      {children}      
+    </div>
+    <div>
+      <div className="grid grid-cols-1 gap-1 pt-4 text-xs text-gray-400 place-items-center">            
+            
+            <div>
```

Comment:
I think the linter is mad because of the empty line and the fact that this block is indented one extra step than it should be.